### ISSUE TITLE
Upper-case UUIDs for file names

### ISSUE THREAD

**Issue by dajoker** [tomboy-notes/tomboy-ng#123]
Tomboy-ng 0.25.2 GTK (Linux) x86_64

Admittedly this is nit-picking, but is there a reason the file names behind -ng are upper-case UUID instead of lower-case ones?  I don't see any value, since they're just hex representations of the UUIDs and those are basically the same regardless of case, so ABCD is the same as abcd, but is there a reason?  All my files from Tomboy (mono) are lower-case, but these new ones are upper case.  I noticed today when running my backup (not the -ng one) and seeing some files with upper-case letters.

**Comment by davidbannon** [comment]
Just the library I used generates uppercase and I guess the Mono ones generate lower case. The rfc4122 does not care -
      hexDigit =
            "0" / "1" / "2" / "3" / "4" / "5" / "6" / "7" / "8" / "9" /
            "a" / "b" / "c" / "d" / "e" / "f" /
            "A" / "B" / "C" / "D" / "E" / "F"
and as we just use then as a filename, apply no other tests after that, I suggest it really does not matter. Early on, I did test Tomboy's ability to open -ng notes, no problem.

ANSWER 2 - I did it that way so I could tell which notes were made in Tomboy and which in -ng.  (If you believe that, I've got this great deal setup with a Nigerian prince, all we need is your bank account details ...)

Davo
